src/main/java/it/gov/pagopa/pu/pagopapayments/endpoint/PaForNodeEndpoint.java

```diff
@@ -111,7 +117,30 @@ public PaSendRTV2Response paSendRTV2(@RequestPayload PaSendRTV2Request request)
     try {
       log.info("processing paSendRTV2 idPA[{}] notice[{}/{}]", request.getIdPA(), request.getReceipt().getFiscalCode(), request.getReceipt().getNoticeNumber());
       PaSendRtDTO paSendRtDTO = paSendRTMapper.paSendRtV2Request2PaSendRtDTO(request);
-      receiptService.processReceivedReceipt(paSendRtDTO);
+      var registryLoggerResult = registryLogger.execute(
+        paSendRtDTO.getFiscalCode(),
+        paSendRtDTO.getIdBrokerPA(),
+        request.getReceipt().getIdPSP(),
+        request.getReceipt().getIdChannel(),
+        request.getReceipt().getPaymentMethod(),
+        null, // @TODO: no ccp in request
+        RegistryEventType.paSendRTV2,
+        request.getReceipt().getNoticeNumber(),
+        paSendRtDTO,
+        () -> {
+          receiptService.processReceivedReceipt(paSendRtDTO);
+          return Triple.of(null, null, RegistryEventOutcome.OK);
+        },
```

Comment:
The registryLogger should return the SOAP response
noticeNumber = nav
```suggestion
      return registryLogger.execute(
        paSendRtDTO.getFiscalCode(),
        paSendRtDTO.getIdBrokerPA(),
        request.getReceipt().getIdPSP(),
        request.getReceipt().getIdChannel(),
        request.getReceipt().getPaymentMethod(),
        null, // @TODO: no ccp in request
        RegistryEventType.paSendRTV2,
        Utilities.nav2Iuv(request.getReceipt().getNoticeNumber()),
        paSendRtDTO,
        () -> {
          receiptService.processReceivedReceipt(paSendRtDTO);
          
          PaSendRTV2Response response = new PaSendRTV2Response();
          response.setOutcome(StOutcome.OK);
        
          return Triple.of(response, null, RegistryEventOutcome.OK);
        },
```